subcommand/server-acl-init/command.go

```diff
@@ -73,8 +81,10 @@ func (c *Command) init() {
 		"Toggle for creating a client agent token")
 	c.flags.BoolVar(&c.flagCreateSyncToken, "create-sync-token", false,
 		"Toggle for creating a catalog sync token")
+	c.flags.BoolVar(&c.flagCreateInjectorToken, "create-injector-namespace-token", false,
```

Comment:
why are you using injector instead of inject here to match the `-create-inject-token` flag?

Reply:
Leftover from before I added `-namespace-` into the flag name.